src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -59,7 +59,7 @@ public static string GenerateJsonSchema(string rulesContent, string allowlistsCo
                     from rule in attributeRules.Rules
                     let ruleInfo = (OpsMetadataRule)rule.ToObject<OpsMetadataRule>()
                     let type = ruleInfo.Type
-                    where ruleInfo.ContentTypes.Contains("conceptual") && type != null && !ruleInfo.Disabled
+                    where ValidContentType(ruleInfo.ContentTypes) && type != null && !ruleInfo.Disabled
```

Comment:
Should be able to remove this filter in this feature.

```suggestion
                    where type != null && !ruleInfo.Disabled
```

Reply:
ok